### PR TITLE
Peagenv2

### PR DESCRIPTION
**Pull request by MichaelDecent**
This commit introduces a new test suite for the direct access methods of api.core and api.core_raw. The tests cover basic CRUD operations, session management, database compatibility, and various edge cases, ensuring robust error handling and functionality for both synchronous and asynchronous operations.

### CODE REVIEW

File: pkgs/standards/autoapi/tests/i9n/test_core_access.py [hunk 1]
```diff
@@ -0,0 +1,586 @@
...
+
+        # Verify user was not persisted
+        with pytest.raises(HTTPException):
+            with next(get_db()) as db:
+                api.core.TestUsersRead(user_id, db)
+
+    @pytest.mark.asyncio
+    async def test_async_db_with_core_via_run_sync(self, async_api):
+        """Test using api.core with async database via run_sync."""
+        api, get_async_db = async_api
```
Comment (cobycloud) [inline pkgs/standards/autoapi/tests/i9n/test_core_access.py:290]: with core calls, use `api.schemas.*` to validate that you can use the corresponding schemas.

core calls include schema validation, whereas core_raw calls do not, and expect dict

**Review by cobycloud — APPROVED** [review]
